include/spdk/bdev.h

```diff
@@ -1733,6 +1733,28 @@ void spdk_bdev_histogram_get(struct spdk_bdev *bdev, struct spdk_histogram_data
 size_t spdk_bdev_get_media_events(struct spdk_bdev_desc *bdev_desc,
 				  struct spdk_bdev_media_event *events, size_t max_events);
 
+enum spdk_bdev_capability_type {
+	/** Bdev module doesn't read or modify data buffer passed in IO request */
+	SPDK_BDEV_CAP_DATA_PASSTHRU = 1u << 0u,
```

Comment:
The description is not correct here. We use it to determine whether bdev supports IOs with mkey extended opt.
And the name should be changed.
Now I don't see any relation between this bdev capability and extended IO opts introduced in the next commits